src/modules/display.js

```diff
@@ -0,0 +1,26 @@
+export default function displayTask(tasks) {
+  const ul = document.getElementById('items');
+  tasks.forEach((task) => {
+    // Creating li to store tasks
+    const li = document.createElement('li');
+    li.draggable = true;
+    li.setAttribute('id', task.index);
+    li.classList.add('d-flex', 'justify-content-between', 'align-items-center');
+    // Creating checkbox
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    li.appendChild(checkbox);
+    // Creating label for checkbox
+    const label = document.createElement('input');
+    label.type = 'text';
+    label.classList.add('flex-grow-1', 'text-left', 'm-1', 'form-control');
+    label.value = task.description;
+    li.appendChild(label);
+    // Creating option icon
+    const icon = document.createElement('i');
+    icon.className = 'fa-solid fa-ellipsis-vertical';
+    li.appendChild(icon);
+    // Appending elements
+    ul.appendChild(li);
+  });
+}
```

Comment:
- Good job done so far, you seem to have done everything right but the tasks are not displaying in the UI on my end, please fix this 🙏 

Your project:
<img width="928" alt="35yw35y" src="https://github.com/batoolfatima2135/To-do-List-App/assets/73607512/101689fc-f9a6-487f-9bb2-c6ad696683ad">